Bottom: 9b8de318d36d1364ba3e26387351aced63cf0bed
Top:    3e5b4cd6ebdacb08a2ba5ede41e07dd127a33d90
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 17:49:54 +0100

GTK HID: Raise already open command window to top with ":" key

Previously, ":" only opened the window, but would not raise it
if already open.




---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 568d46e..09d8025 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1758,7 +1758,7 @@ LibraryChanged (int argc, char **argv, int x, int y)
 static int
 Command (int argc, char **argv, int x, int y)
 {
-  ghid_handle_user_command (FALSE);
+  ghid_handle_user_command (TRUE);
   return 0;
 }
